### PR TITLE
Add partial update test case

### PR DESCRIPTION
**Pull request by loay**

### CODE REVIEW

**Comment by loay** [comment]
@bajtos PTAL. Thanks. 
Reference https://github.com/strongloop/loopback/pull/2693#r77506564

File: test/user.test.js [hunk 1]
```diff
@@ -2124,6 +2124,40 @@ describe('User', function() {
         },
       ], done);
     });
+
+    it('keeps sessions AS IS if partial update is done with age', function(done) {
```
Comment (bajtos) [inline test/user.test.js]: Please move the test closer to existing `keeps sessions AS IS` tests and rename it to mention `updateAll`, e.g. `keeps sessions AS IS if non-email property is changed using updateAll`.

**Comment by loay** [comment]
test please

**Comment by loay** [comment]
test please

**Comment by loay** [comment]
@bajtos PTAL Thanks

File: test/user.test.js [hunk 2]
```diff
@@ -1977,6 +1977,43 @@ describe('User', function() {
         ], done);
       });
 
+      it('keeps sessions AS IS if non-email property is changed using updateAll', function(done) {
+        var userPartial;
+        async.series([
+          function createPartialUser(next) {
+            User.create(
+                   { email: 'partial@example.com', password: 'pass1', age: 25 },
```
Comment (bajtos) [inline test/user.test.js]: Nitpick: use two spaces for indentation. I am surprised `eslint` is not complaining.

**Pull request merged**